app/src/main/java/com/globant/example/mentorapp/home/domain/model/UserEntity.java

```diff
@@ -0,0 +1,124 @@
+package com.globant.example.mentorapp.home.domain.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class UserEntity {
+
+    @Expose
```

Comment:
Expose annotation's purpose is implemented by GsonBuilder.excludeFieldsWithoutExposeAnnotation but you are not using that Builder. Why did you declare these attributes like Expose?